parsimonious/grammar.py

```diff
@@ -95,8 +87,9 @@ def _copy(self):
 
         """
         new = Grammar.__new__(Grammar)
+        super(Grammar, new).__init__(self.items())
```

Comment:
No, since `Grammar.__init__` doesn't take that type as input. Note that I'm actually calling the superclass of (`OrderedDict.__new__(Grammar)`) to construct a new `Grammar` instance. We can't pass the rules as kwargs, since those aren't ordered either (prior to Python 3.6). 

I definitely agree this is pretty ugly, and involves duplicating some of the implementation of `Grammar.__init__`, so I'd love a better solution here.

The other option I considered was adding a new parameter to pass an iterable of rules. That seems undesirable since it's complicating the API of Grammar with no use case outside of this private method.